scp/tickets.php

```diff
@@ -201,9 +201,10 @@
                 if(!$ticket->checkStaffPerm($thisstaff))
                     $ticket=null;
             } elseif(!$errors['err']) {
-                $errors['err']=sprintf(
-                    __('Unable to update %s. Correct any errors below and try again.'),
-                    __('ticket'));
+                $errors['err']=sprintf('%s %s',
+                    __('Unable to update %s.'),
+                    __('ticket'),
+                    __('Correct any errors below and try again.'));
```

Comment:
This looks like a mistake. The `%s` in the first replacement will not be replaced. I think it should be
```php
sprintf('%s %s',
    sprintf(__('Unable to update %s.'), __('this ticket')),
    __('Correct any errors below and try again.')
);
```